stdlib/public/core/CTypes.swift

```diff
@@ -101,6 +101,11 @@ public typealias CLongDouble = Double
 #endif
 #elseif os(OpenBSD)
 public typealias CLongDouble = Float80
+#elseif os(FreeBSD)
+#if arch(x86_64) || arch(i386)
```

Comment:
My preference would be to swap the architectures here, but that is low priority.

Reply:
Happy to—I was just matching the Darwin and Linux branches.